packages/frontend/app/lib/api/asset.server.ts

```
@@ -139,8 +140,12 @@ export const listAssets = async (request: Request, args: QueryAssetsArgs) => {
   return response.data.assets
 }
 
-export const createAsset = async (request: Request, args: CreateAssetInput) => {
-  const apolloClient = await getApolloClient(request)
+export const createAsset = async (
+  request: Request,
+  args: CreateAssetInput,
+  tenantId?: string
+) => {
+  const apolloClient = await getApolloClient(request, tenantId)
```

I think something is missing here.

The `tenantId` is never passed in `createAsset` (just called once) nor `getApolloClient` (only passed in here but its `undefined`). And there is no `tenantId` on the `CreateAssetInput`. The backend mutation uses the header, so it's just always going to be the operator.

You can validate this by changing the `defaultValue` on the tenant dropdown to any bogus value and submitting and see that it works (should fail because its not a real tenant id).

I think the intention was probably to pass the `tenantId` in here and override the header but I think previously we agreed we didn't want to do that to ensure the header always identifies the requestor. 

I think we can remove the header overriding, put the `tenantId` on the gql input, and use the `ctx.forTenantId` provided by the [setForTenantIdGraphQLMutationMiddleware](https://github.com/interledger/rafiki/blob/2893/multi-tenancy-v1/packages/backend/src/graphql/middleware/index.ts#L55) (looks like mutation context type for assets needs to be updated to `ForTenantIdContext`).